appyters/dimensionality_reduction_visualization/appyter.json

```diff
@@ -1,23 +1,25 @@
 {
   "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/main/schema/appyter-validator.json",
   "name": "dimensionality_reduction_visualization",
-  "title": "Dimensionality_reduction_visualization",
+  "title": "PCA, t-SNE, UMAP Appyter",
   "version": "0.0.1",
-  "description": "An appyter for to perform visualization using PCA, t-SNE, and UMAP methods.",
+  "description": "An Appyter to perform visualization of a data matrix using PCA, t-SNE, and UMAP methods.",
   "image": "scatter_screenshot.png",
   "authors": [
     {
       "name": "Juncheng Pan",
       "email": "pan_juncheng@yahoo.com"
     }
   ],
-  "url": "https://github.com/MaayanLab/appyter-catalog",
+  "url": "https://github.com/MaayanLab/appyter-catalog/tree/main/appyters/dimensionality_reduction_visualization",
   "tags": [
     "React",
     "react-scatter-board",
-    "PCA",
-    "UMAP",
-    "t_SNE"
+    "bokeh",
+    "3D",
+    "interactive",
+    "downloadable",
+    "MulticoreTSNE"
```

Comment:
Not sure about these tags but frankly I don't have better ideas. Maybe:
- Exploration
- Dimensionality Reduction
- Visualization

These would be keywords a user searches for with the appyter catalog search box. No one is looking for MutlicoreTSNE or bokeh, those are implementation details. Same thing with react/react-scatter-board.

Reply:
Removed some less general tags